docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -408,11 +445,21 @@ This section contains a native hashtable of all generic method instantiations co
 
 ## READYTORUN_SECTION_MANIFEST_METADATA
 
-Manifest metadata is an [ECMA-335] metadata blob containing extra reference assemblies within the version bubble introduced by inlining on top of assembly references stored in the input MSIL. As of R2R version 3.1, the metadata is only searched for the AssemblyRef table. This is used to translate module override indices in signatures to the actual reference modules (using either the `READYTORUN_FIXUP_ModuleOverride` bit flag on the signature fixup byte or the `ELEMENT_TYPE_MODULE_ZAPSIG` COR element type).
+Manifest metadata is an [ECMA-335] metadata blob containing extra reference assemblies within
+the version bubble introduced by inlining on top of assembly references stored in the input MSIL.
+As of R2R version 3.1, the metadata is only searched for the AssemblyRef table. This is used to
+translate module override indices in signatures to the actual reference modules (using either
+the `READYTORUN_FIXUP_ModuleOverride` bit flag on the signature fixup byte or the
+`ELEMENT_TYPE_MODULE_ZAPSIG` COR element type).
 
-**Disclaimer:** The manifest metadata is a new feature that hasn't shipped yet; it involves straightforward adaptation of a fragile nGen technology to ReadyToRun images as an expedite means for enabling new functionality (larger version bubble support). The precise details of this encoding are still work in progress and likely to further evolve.
+**Disclaimer:** The manifest metadata is a new feature that hasn't shipped yet; it involves
```

Comment:
The note that this evolved from fragile NGen can be in design notes. It should not be in the file format spec. Many other things in the file format spec evolved from fragile NGen, but we do not mention it anywhere.

Reply:
Moved to design notes in 2nd commit.